models/wise-complete-policy.go

```diff
@@ -59,6 +62,44 @@ func (wisePolicy *WiseCompletePolicy) ToDomain() Policy {
 	return policy
 }
 
+func updatePolicyEndDate(policy *Policy, wisePolicy *WiseCompletePolicy) {
+	location, err := time.LoadLocation("Europe/Rome")
+	lib.CheckError(err)
+	policyEndDate := wisePolicy.Contract.PolicyExpirationDate.In(location)
+	_, offset := policyEndDate.Zone()
+	policy.EndDate = policyEndDate.Add(time.Duration(time.Second * time.Duration(offset)))
+}
+
+func updatePolicyPrice(policy *Policy, wisePolicy *WiseCompletePolicy) {
+	policy.PriceGross = wisePolicy.Contract.AnnualGrossPrice
+	policy.PriceNett = wisePolicy.Contract.NetAmount
+	policy.TaxAmount = policy.PriceGross - policy.PriceNett
+
+	if wisePolicy.Contract.InstalmentTypeCode == WiseYearlyPaymentSplitCode {
+		policy.PaymentSplit = string(PaySplitYear)
+		return
+	}
+
+	policy.PriceGrossMonthly = wisePolicy.Contract.GrossAmount
+	policy.PriceNettMonthly = wisePolicy.Contract.NetAmount
+	policy.TaxAmountMonthly = policy.PriceGrossMonthly - policy.PriceNettMonthly
+
+	if wisePolicy.Contract.InstalmentTypeCode == WiseMonthlyPaymentSplitCode {
+		policy.PaymentSplit = string(PaySplitMonthly)
+		policy.PriceGross = policy.PriceGrossMonthly * 12
+		policy.PriceNett = policy.PriceNett * 12
+		policy.TaxAmount = policy.TaxAmountMonthly * 12
+		return
+	}
+	if wisePolicy.Contract.InstalmentTypeCode == WiseSemestralPaymentSplitCode {
+		policy.PaymentSplit = string(PaySplitSemestral)
+		policy.PriceGross = policy.PriceGrossMonthly * 2
+		policy.PriceNett = policy.PriceNett * 2
+		policy.TaxAmount = policy.TaxAmountMonthly * 2
```

Comment:
Is this logic accurate?
I'd expect monthly * 6 or yearly / 2

Reply:
Yes, because we are saving the `*Semestral` price also in the `*Monthly` prices. Me and @Yousef-Hammar have discussed that it makes more sense to use the `PriceGross` as the annual value and `PriceGrossMonthly` as the related `PaymentSplit` value, be it monthly, semestral or singleInstallment. The thing that we need to do as a next step would be renaming the `*Monthly` prices to something more fitting. Do you agree @diogo-wopta?

Reply:
And also I don't think it makes sense to have a field for each `PaymentSplit` like, `PriceGrossSingleInstallment`, `PriceGrossSemestral`.

Reply:
Ok. A few options are:
PriceGrossSplit / PriceGrossRate / PriceGrossInstallment